src/app/components/wrapper-components/online-game-wrapper/online-game-wrapper.component.spec.ts

```diff
@@ -41,7 +43,10 @@ describe('OnlineGameWrapper for non-existing game', () => {
         tick(1);
 
         // Then it goes to /notFound with the expected error message
-        expectValidRouting(router, ['/notFound', GameWrapperMessages.NO_MATCHING_GAME('invalid-game')], NotFoundComponent, { skipLocationChange: true });
+        const expectedRoute: string[] = ['/notFound', GameWrapperMessages.NO_MATCHING_GAME('invalid-game')];
+        expectValidRouting(router, expectedRoute, NotFoundComponent, { skipLocationChange: true });
+
+        discardPeriodicTasks();
```

Comment:
c'pas un tick(3000) ici simplement ?

Reply:
unfortunately not! tick(3000) ou tick(300000), rien n'y fait. Les periodic task sont des setInterval, probablement ajoutés par notre code, mais impossible de trouver d'où elles viennent ni comment les supprimer; `discardPeriodicTasks()` dit à karma de toutes les supprimer.

Reply:
ça soulève des questions quand même.
Et quid de discardPeriodicTasks vs flush ?

Reply:
flush enlève les timers non-périodiques, discardPeriodicTask est l'équivalent pour les périodiques. En gros setTimeout (non-périodique) vs. setInterval (périodique)